### PR TITLE
media-video/obs-studio: update CEF and add version 27.1.1 with EAPI-8

### PR DESCRIPTION
**Pull request by Chiitoo**

### CODE REVIEW

**Review by ionenwks — COMMENTED** [review]
Just a few nitpicks and deps tightening, lgtm otherwise.

File: media-video/obs-studio/obs-studio-27.1.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+
+	if ! has_version "sys-apps/dbus"; then
+		elog
+		elog "The 'sys-apps/dbus' package is not installed, but"
+		elog "could be used for disabling hibernating, screensaving,"
+		elog "and sleeping.  Where it is not installed,"
+		elog "'xdg-screensaver reset' is used instead"
+		elog "(if 'x11-misc/xdg-utils' is installed)."
+		elog
+	fi
```
[the remark this answers is not in the document]
Comment (Chiitoo) [reply media-video/obs-studio/obs-studio-27.1.1.ebuild]: Ah, right... it looks like the dependency was added in 2a66f9175c3 [1], and I didn't notice/think of that at the time...

I guess I'll remove the 'if', and maybe make the thing better optional at some point.

1. https://gitweb.gentoo.org/repo/gentoo.git/commit/?id=2a66f9175c3378079f295971d31ad8f863138c30

File: media-video/obs-studio/obs-studio-27.1.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+		-DBUILD_BROWSER=$(usex browser)
+		-DBUILD_VST=no
+		-DENABLE_WAYLAND=$(usex wayland)
+		-DDISABLE_ALSA=$(usex !alsa)
+		-DDISABLE_DECKLINK=$(usex !decklink)
+		-DDISABLE_FREETYPE=$(usex !truetype)
+		-DDISABLE_JACK=$(usex !jack)
+		-DDISABLE_LIBFDK=$(usex !fdk)
+		-DENABLE_PIPEWIRE=$(usex pipewire)
+		-DDISABLE_PULSEAUDIO=$(usex !pulseaudio)
```
[the remark this answers is not in the document]
Comment (Chiitoo) [reply media-video/obs-studio/obs-studio-27.1.1.ebuild:150]: I'll add glib and libglvnd, and indeed, take a better look at the rest at a better time if that's okay.

File: media-video/obs-studio/obs-studio-27.1.1.ebuild [hunk 3]
```diff
@@ -0,0 +1,218 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+CMAKE_REMOVE_MODULES_LIST=( FindFreetype )
+LUA_COMPAT=( luajit )
+PYTHON_COMPAT=( python3_{8..10} )
+
+inherit xdg cmake lua-single python-single-r1
```
[the remark this answers is not in the document]
Comment (Chiitoo) [reply media-video/obs-studio/obs-studio-27.1.1.ebuild]: Done.

File: media-video/obs-studio/obs-studio-27.1.1.ebuild [hunk 4]
```diff
@@ -0,0 +1,218 @@
...
+	virtual/udev
+	x11-libs/libX11
+	x11-libs/libXcomposite
+	x11-libs/libXfixes
+	x11-libs/libXinerama
+	x11-libs/libXrandr
+	x11-libs/libxcb:=
+	alsa? ( media-libs/alsa-lib )
+	browser? (
+		app-accessibility/at-spi2-atk
```
[the remark this answers is not in the document]
Comment (Chiitoo) [reply media-video/obs-studio/obs-studio-27.1.1.ebuild:74]: Done.

**Pull request closed without merging**